src/openrct2/Game.cpp

```diff
@@ -532,7 +532,8 @@ void reset_all_sprite_quadrant_placements()
 
 void save_game()
 {
-    if (!gFirstTimeSaving)
+    auto name = Path::GetFileNameWithoutExtension(gScenarioSavePath);
+    if (!gFirstTimeSaving && name.rfind("autosave_", 0) != 0)
```

Comment:
I’d rather this check was carried out upon _loading_ an autosave.

Reply:
We could store a flag in the autosave file to say that it was an autosave rather than relying on the file name.

Reply:
I'm open to change the implementation, but new to C++ and OpenRCT2. To prevent me going the wrong way, could you give a hint to where / how this flag should be added? Thanks!

Reply:
You would read/write it to a global field somewhere in ParkFile.cpp.

Reply:
I haven't figured out how to do it yet unfortunately. Will give it a go later again, if I still can't get it done I'll ask for more help.

Reply:
@IntelOrca @Gymnasiast Resumed this one today and think I worked things out better now. Could you re-review? Thanks!